spec/defines/vhost_spec.rb

```diff
@@ -1287,6 +1287,27 @@
             ).with_content(%r{## Proxy rules})
           }
         end
+
+        context 'proxy_dest_reverse_match' do
+          let :params do
+            {
+              'docroot' => '/rspec/docroot',
+              'proxy_pass_match' => [
+                {
+                  'path'     => '/',
+                  'url'      => 'http://localhost:8081/',
+                },
+              ],
+            }
+          end
+
+          it {
+            is_expected.to contain_concat__fragment('rspec.example.com-proxy').with_content(
+              %r{ProxyPassReverse / http:\/\/localhost:8081/\/},
```

Comment:
I think the escaping is redundant since you use `%r{}`.

Though I do wonder if the double `/` at the end is intended. Should it actually match a complete line?
```suggestion
              %r{^\s*ProxyPassReverse / http://localhost:8081//$},
```